Cargo.toml

```diff
@@ -1,17 +1,15 @@
 [package]
 name = "landlock"
-version = "0.1.0"
-authors = [
-	"Mickaël Salaün <mic@digikod.net>",
-	"Vincent Dagonneau <vincentdagonneau@gmail.com>"
-]
-edition = "2018"
+version = "0.2.0"
+edition = "2021"
 rust-version = "1.63"
+authors = ["Mickaël Salaün <mic@digikod.net>"]
```

Comment:
The authors field has been made optional: https://rust-lang.github.io/rfcs/3052-optional-authors-field.html

> The crates.io registry does not allow users to change the contents of already published versions: this is highly desirable to ensure working builds don't break in the future, but it also has the unfortunate side-effect of preventing people from updating the list of crate authors defined in Cargo.toml's package.authors field.
>
> This is especially problematic when people change their name or want to remove their name from the Internet, and the crates.io team doesn't have any way to address that at the moment except for deleting the affected crates or versions altogether. We don't do that lightly, but there were a few cases where we were forced to do so.
>
> The contents of the field also tend to scale poorly as the size of a project grows, with projects either making the field useless by just stating "The $PROJECT developers" or only naming the original authors without mentioning other major contributors.

Reply:
Good catch, I'll remove this field.